Makefile

```diff
@@ -230,6 +230,10 @@ install-zetae2e: go.sum
 	@go install -mod=readonly $(BUILD_FLAGS) ./cmd/zetae2e
 .PHONY: install-zetae2e
 
+solana:
+	@echo "Building solana docker image"
+	$(DOCKER) build -t solana-local -f contrib/localnet/solana/Dockerfile contrib/localnet/solana/
```

Comment:
Why having a special makefile command, isn't it part of the default localnet?

Reply:
The solana local validator is quite CPU intensive and should not run by default.

The solana tests will run separately so it should be built separately?